### PR TITLE
fix: repo qa

### PR DESCRIPTION
**Pull request by AlexPatrie**
_**What does this PR do?**_:
- Adjusts styles across repo components.
- QA as needed.

### CODE REVIEW

File: apps/dispatch/src/app/components/home/home.component.html [hunk 1]
```diff
@@ -1,5 +1,5 @@
-<div class="home-container-run">
-  <mat-card class="description-run">
+<div *ngIf="!isMobileHomeDispatch; else mobileViewDispatchHome" class="home-container run">
```
Comment (jcschaff) [inline apps/dispatch/src/app/components/home/home.component.html]: your scss file refers to a 'home-container-run' but here you use 'home-container' and 'run' selectors.

**Review by jcschaff — COMMENTED** [review]
see comment about scss selectors

File: apps/dispatch/src/app/components/home/home.component.html (hunk 1 above)
Comment (AlexPatrie) [reply apps/dispatch/src/app/components/home/home.component.html]: I appreciate having an extra set of eyes for something like this!

**Review by jcschaff — APPROVED** [review]
looks good, comment about css selector has been resolved.

**Pull request merged**